t/issue32.t

```diff
@@ -7,12 +7,12 @@ use HTTP::Request;
 use HTTP::Response;
 
 my $req  = HTTP::Request->new(GET => "http://example.com");
-my $resp = HTTP::Response->new(200, 'OK', ['Set-Cookie', q!a="b;c;\\"d"; expires=Fri, 06-Nov-2025 08:58:34 GMT; domain=example.com; path=/!]);
+my $resp = HTTP::Response->new(200, 'OK', ['Set-Cookie', q!a="b;c;\\"d"; expires=Fri, 06-Nov-2999 08:58:34 GMT; domain=example.com; path=/!]);
```

Comment:
Will this cause failures on systems with a year 2038 problem?

Reply:
Good question. I now tested that tests still pass on i586 even in 2037.